metastats-application/src/main/java/life/qbic/metastats/cli/MetaStatsTool.java

```diff
@@ -26,13 +34,41 @@ public MetaStatsTool(final MetaStatsCommand command) {
     public void execute() {
         // get the parsed command-line arguments
         final MetaStatsCommand command = super.getCommand();
-        /**OpenBisSession session =
-        DatabaseGateway db = new OpenBisSearch(IApplicationServerApi,"")
-        ProjectSpecification spec = new RequestExperimentData()
+        // get properties
+        LOG.debug(command.conf);
+        JsonParser experimentProps = new JsonParser(command.conf);
+        Map credentials = experimentProps.parse();
 
-        MetaStatsController msc = new MetaStatsController(command.conf,command.projectCode);*/
+        JsonValidator validator = new JsonValidator("/model.schema.json");
 
-    }
+        //define output classes
+        MSMetadataPackageOutput metaStatsPresenter = new MetaStatsPresenter(new TSVFileCreator());
+
+        experimentProps = new JsonParser("openbisToMetastatsExperiment.json");
+        Map expMappingInfo = experimentProps.parseStream();
+
+        JsonParser sampleProps = new JsonParser("openbisToMetastatsSample.json");
+        Map sampleMappingInfo = sampleProps.parseStream();
+
+        PropertiesMapper mapper = new OpenBisMapper(expMappingInfo, sampleMappingInfo);
 
-    // TODO: override the shutdown() method if you are implementing a daemon and want to take advantage of a shutdown hook for clean-up tasks
+        //define use case
+        FilterExperimentData filter = new FilterExperimentDataImpl(metaStatsPresenter,mapper,validator);
```

Comment:
```suggestion
        FilterExperimentData filterExperimentData = new FilterExperimentDataImpl(metaStatsPresenter,mapper,validator);
```
It is not a very long variable name and describing what you are filtering by naming your use case variable might be a good idea.